metricsrouterv3/metrics_router_v3.go

```diff
@@ -114,6 +114,10 @@ func GetServiceURLForRegion(region string) (string, error) {
 	var endpoints = map[string]string{
 		"au-syd":           "https://au-syd.metrics-router.cloud.ibm.com/api/v3",           // The server for IBM Cloud Metrics Routing Service in the au-syd region.
 		"private.au-syd":   "https://private.au-syd.metrics-router.cloud.ibm.com/api/v3",   // The server for IBM Cloud Metrics Routing Service in the au-syd region.
+		"br-sao":           "https://br-sao.metrics-router.cloud.ibm.com/api/v3",           // The server for IBM Cloud Metrics Routing Service in the br-sao region.
```

Comment:
Did you manually update the generated code, or did you re-generate the code with the SDK generator and an updated API definition?
If the former, then please re-do your PR by re-generating the code using the updated API definition.

Reply:
No I have generated using SDK generator